admin-dev/themes/default/js/vendor/OverlayScrollbars.js

```diff
@@ -0,0 +1,6222 @@
+/*!
+ * OverlayScrollbars
+ * https://github.com/KingSora/OverlayScrollbars
+ *
+ * Version: 1.6.3
+ *
+ * Copyright KingSora.
+ * https://github.com/KingSora
+ *
+ * Released under the MIT license.
+ * Date: 31.01.2019
+ */
+
```

Comment:
I can see that you're inclined into adding heavy jquery plugins for simple tasks. I think you need to understand that performance should always prime over "let's make this little thing a bit prettier by adding a xx+ KB jquery plugin to be loaded on each page". 
If you really want to make a prettier scrollbar, just use [::-webkit-scrollbar](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar) to style it for webkit browsers.

Reply:
The plugin has been set up to fix a bug on ie11 and edge.
It was already planned to use 'http://utatti.github.io/perfect-scrollbar/' but I used this one.
"::-webkit-scrollbar" doesn't work for ie11.

Reply:
It was proposed it in #9588, indeed, but [perfect-scrollbar ](https://github.com/utatti/perfect-scrollbar/blob/master/dist/perfect-scrollbar.min.js) weights only 17.9 KB while your solution is much heavier (you added the unminified 330 KB version). Styling the scrollbar by css may indeed not work in all browsers, but virtually adds little to no additional weight to the backoffice, which is a win.

Reply:
At the very least, if you want to use this plugin, you should add the minified version.